Mako-IoT.Device.sln.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
Please remove this file from PR. Also you can add it to gitignore

Reply:
Did you accidentally forget to push changes? File is still there.

Reply:
I see the file being removed in this commit:

https://github.com/CShark-Hub/Mako-IoT.Device/pull/18/commits/6ec7dae21d6daa887f7a58d7cee1f20378b887cf

And the .gitignore being updated in the next commit:

https://github.com/CShark-Hub/Mako-IoT.Device/pull/18/commits/e126af97f50d303685d98686bf6782fa2050c02d